lib/partials.ts

```diff
@@ -167,8 +167,8 @@ hb.registerHelper('import', async (options) => {
 			const safeContent = new hb.SafeString(
 				partialContent.slice(0, partialContent.length - 1),
 			);
-			const builtContent = await hb.compile(safeContent.toString())(
-				options.data.root,
+			const builtContent = await Promise.resolve(
+				hb.compile(safeContent.toString())(options.data.root),
 			);
```

Comment:
I don't see any reason to go via an `await Promise.resolve` indirection here? If I'm correct I'd much prefer
```suggestion
			const builtContent = hb.compile(safeContent.toString())(options.data.root);
```

Reply:
Improved the types for promised-handlebars and now the Promise.resolve is not needed

Reply:
Ah that makes sense, and definitely much better to fix at the typings level 👍